core/src/main/java/com/flipkart/gap/usl/core/config/v2/ApplicationConfiguration.java

```diff
@@ -10,19 +10,23 @@
 import lombok.Data;
 
 import java.io.Serializable;
+import java.util.List;
 import java.util.Map;
 
 /**
  * Created by piyush.chhabra on 25/03/2019
  */
 @Data
 public class ApplicationConfiguration implements Serializable {
+
+    private EventProcessorConfig externalKafkaConfig;
```

Comment:
it is better to create separate config class  for external kafka as event processor config may have other configs than kafka

Reply:
why are still keeping it here, why cant we create separate ApplicationConfiguration class for new job and put external kafka configuration there?

Reply:
i see there is a separate config down there, if its not used please detle